content/en/docs/tasks/network/validate-dual-stack.md

```diff
@@ -14,7 +14,7 @@ This document shares how to validate IPv4/IPv6 dual-stack enabled Kubernetes clu
 {{% capture prerequisites %}}
 
 * Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
-* Kubenet network plugin
+* A network plugin that supports dual-stack (such as Kubenet or Calico)
```

Comment:
```suggestion
* A [network plugin](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) that supports dual-stack (such as Kubenet or Calico)
```
?

I notice that https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/ is marked as having gone alpha in v1.17 - it seems like that page also wants a tweak.

Reply:
I guess you're right about that network plugins page, but I'd rather leave that to someone else as I feel less personally confident about it.

Reply:
No worries @neiljerram, I can probably fix that.